### DISCUSSION TITLE
More documentations on how to use Positron effectively for R and Python project

### DISCUSSION THREAD

**Discussion by ntluong95** [posit-dev/positron#4084] [Feedback]
There are several blog posts elsewhere on the internet where people share how to setup Positron for R and Python project. But it might be good if we can have one in Wiki page (potentially official page later) share how we can use Positron effectively. Some ideas to consider:

- You are an R user from Rstudio want to learn new IDE, new language. What are similar, what are different?
- You are an user from VSCode, so what make Positron different?
- How to manage multiple versions of R and Python, usage cases?

**Comment by juliasilge** [comment]
Thanks for these suggestions! We'll probably wait until we start our official documentation site (which is a near term project, not very far off).
